target/linux/generic/hack-5.15/726-net-eth-dpaa2-eth-do-not-hold-rtnl_lock.patch

```diff
@@ -62,7 +62,7 @@ Signed-off-by: Russell King <rmk+kernel@armlinux.org.uk>
  	if (err) {
  		netdev_err(net_dev, "phylink_fwnode_phy_connect() = %d\n", err);
  		goto err_phylink_destroy;
-@@ -372,7 +374,9 @@ void dpaa2_mac_disconnect(struct dpaa2_m
+@@ -378,7 +380,9 @@ void dpaa2_mac_disconnect(struct dpaa2_m
```

Comment:
Please refresh the mediatek target after this change too. The Ci failed with:
````
Run . .github/workflows/scripts/ci_helpers.sh
target/linux/mediatek/patches-5.15/730-v6.5-net-phy-add-driver-for-MediaTek-SoC-built-in-GE-PHYs.patch
Kernel patches for mediatek/filogic require refresh. (run 'make target/linux/refresh' and force push this pr)
You can also check the provided artifacts with the refreshed patch from this CI run.
Error: Process completed with exit code 1.
````

Reply:
This was an unrelated problem in the openwrt-23.05 branch fixed in 65c1f418e3add2d32b036696530dffddaba64d72